src/backend/stream.rs

```diff
@@ -0,0 +1,821 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use backend::*;
+use backend::cork_state::CorkState;
+use cubeb;
+use libpulse_sys::*;
+use std::ptr;
+use libc::{c_char,c_void};
+
+const PULSE_NO_GAIN: f32 = -1.0;
+const PA_USEC_PER_MSEC: pa_usec_t = 1000;
+const PA_USEC_PER_SEC: pa_usec_t = 1000000;
```

Comment:
I remember why I did this initially. I was using the libpulse-sys crate that was missing these values. Now that has been replaced, I'll add the definitions to pulse-ffi crate.